docs/middleware/ics29-fee/msgs.md

```diff
@@ -0,0 +1,64 @@
+<!--
+order: 3
+-->
+
+# Escrowing and paying out fees
```

Comment:
Should this section be titled just `Escrowing fees`? Since I think there is only a reference to the messages to escrow fees here and the part about paying out fees is talked about in the next section.